examples/cross-namespace-routing/README.md

```diff
@@ -1,13 +1,13 @@
 # Example
 
-In this example, we expand on the simple [cafe-example](../cafe-example) by using a ReferenceGrant to route to backends
+In this example, we expand on the simple [cafe-example](../cafe-example) by using a `ReferenceGrant` to route to backends
```

Comment:
Do we need back ticks here? I thought we decided not to use them for Gateway API resources

Reply:
I wasn't sure what the pattern was. It seemed like these terms were emphasized at certain points of the doc (like here being the main topic of the doc). But I can remove all instances like this for consistency.